src/routes/user/userComponents/userData.svelte

```diff
@@ -26,35 +168,126 @@
                                     class="form--disabled input variant-form-material"
                                     type="text"
                                     name="inputField"
-                                    placeholder={userInfo[i]}
-                                    id="input_{key}"
+                                    placeholder={personalData.lastName}
+                                    bind:this={lastNameReference}
+                                />
+                            </label>
+                        </form>
+                    </td>
+                </tr>
+                <tr>
+                    <th class="text-left"> Telefonnummer: </th>
+                    <td>
+                        <form id="form">
+                            <label class="label">
+                                <input
+                                    disabled={!formEditEnabled}
+                                    class="form--disabled input variant-form-material"
+                                    type="text"
+                                    name="inputField"
+                                    placeholder={personalData.phoneNumber}
+                                    bind:this={phonenumberReference}
+                                />
+                            </label>
+                        </form>
+                    </td>
+                </tr>
+                <tr>
+                    <th class="text-left"> Straße: </th>
```

Comment:
```suggestion
                    <th class="text-left">Straße:</th>
```
Oder waren die Spaces da mit Absicht?